src/component/WorkSched.js

```diff
@@ -10,12 +10,12 @@ class WorkSched extends Component {
                     <div>
                         <Container>
                             <Row>
```

Comment:
Can we add loop to this component?

Reply:
@atherdon I cannot understand how adding a loop here will be useful, or how even we will be able to add loop here, because there is no repetitive task as such.

Reply:
my main idea that we have a similar items inside. so i think we should make it more programable

Reply:
@atherdon Okay I have made the changes, please check and comment!